mitodl/open-discussions
# Parse bootcamps json data into Course model

#### What are the relevant tickets?
#1976 

#### What's this PR do?
This PR adds a framework for the bootcamps.json data, as well as functions to parse that data into the course_catalog Course table.

#### How should this be manually tested?
`course_catalog.tasks.get_bootcamp_data` can be run manually. You can then check the database to see if all 3 existing bootcamps were properly loaded.
